core/src/main/java/contacts/core/Where.kt

```diff
@@ -582,119 +540,75 @@ internal fun <T : AbstractDataField> Where<T>.inRawContactsTable(): Where<RawCon
  *  With this in mind, we can do some cool stuff like in [copyWithNewFieldType]!
  */
 class Where<out T : Field> private constructor(
-    private val lhs: LeftHandSide,
-    private val operator: Operator,
-    private val rhs: RightHandSide,
+    internal val lhs: LeftHandSide,
+    internal val operator: Operator,
+    internal val rhs: RightHandSide,
```

Comment:
Had to make these internal to avoid having to write long functions inside this class.